holy-grail-frontend/src/features/Account/AccountPage.tsx

```diff
@@ -35,15 +44,28 @@ export const AccountPage = () => {
     }
   }, [isLoading, user]);
 
+  useEffect(() => {
+    if (!isDesktop) {
+      setTitle('Edit Account');
+      setSubtitle('View, edit and update your account here!');
+    } else {
+      setTitle('Account details');
+      setSubtitle('View your account details here!');
+      setRenderMenuType(<AccountDetails changeEmailClick={changeEmailClick} />);
+    }
+  }, [isDesktop]);
+
   const navProps: VerticalNavProps[] = [
     {
       icon: AccountBoxIcon,
       label: 'Account details',
       onClick: () => {
         setTitle('Account details');
-        setSubtitle('Change and update your account details here!');
-        setRenderMenuType(<AccountDetails />);
+        setSubtitle('View your account details here!');
+        setRenderMenuType(<AccountDetails changeEmailClick={changeEmailClick} />);
+        setActiveElem([true, false, false]);
```

Comment:
Is this bad practice? in terms of scalability and maintainability. although it works, having a list of booleans is generally a bad idea, and i was thinking to use a mapping instead.